privacy_evaluator/output/user_output.py

```diff
@@ -1,103 +1,43 @@
+import json
 import numpy as np
-import pandas as pd
-from matplotlib import pyplot as plt
-
-from typing import Tuple
 
 
 class UserOutput:
+    def _to_json(self, filter: np.ndarray = None) -> str:
```

Comment:
Should that be a private method? Otherwise, you can remove the underscore in from of it.

Reply:
done